src/DataStructures/MakeWithValue.hpp

```diff
@@ -7,6 +7,7 @@
 #pragma once
```

Comment:
Not a request at all, but want to bring it up for discussion.

Since TaggedTuple is in Utilities and we normally have the MakeWithValue impls inside the header of the data structure it makes, we run into an odd dependency. I don't think TaggedTuple should be moved to DataStructures, since it's very general and low level, though it is technically a data structure. However, I have nothing against moving MakeWithValue to Utilities, and then having the TaggedTuple implementation appended to it. I just don't think anything in Utilities should depend on things in DataStructures (if possible). Another possibility is to leave the impl for TaggedTuple and possibly other Utilities data structures in MakeWithValue.hpp, or to add another header that holds all the impls.

Reply:
Discussion with @kidder and @markscheel with agreement to make an issue and move MakeWithValue.hpp to Utilities later